src/test/java/com/puppycrawl/tools/checkstyle/checks/OrderedPropertiesCheckTest.java

```diff
@@ -0,0 +1,184 @@
+        final DefaultConfiguration checkConfig = createModuleConfig(OrderedPropertiesCheck.class);
+        final OrderedPropertiesCheck check = new OrderedPropertiesCheck();
+        check.configure(checkConfig);
+        final String fileName =
+                getPath("InputOrderedPropertiesCheckNotExisting.properties");
+        final File file = new File(fileName);
+        final FileText fileText = new FileText(file, Collections.emptyList());
+        final SortedSet<LocalizedMessage> messages =
+                check.process(file, fileText);
+        assertEquals("Wrong messages count: " + messages.size(),
+                1, messages.size());
+        final LocalizedMessage message = messages.iterator().next();
+        final String retrievedMessage = messages.iterator().next().getKey();
+        assertEquals("Message key '" + retrievedMessage
+                        + "' is not valid", "unable.open.cause",
+                retrievedMessage);
+        assertEquals("Message '" + message.getMessage()
+                        + "' is not valid", message.getMessage(),
+                getCheckMessage(MSG_IO_EXCEPTION_KEY, fileName, getFileNotFoundDetail(file)));
+    }
+
+    /**
+     * This test validates the PIT mutation of getIndex().
+     * Here the for statement for
+     * (int index = startLineNo; index < fileText.size(); index++)
+     * will change to
+     * for (int index = startLineNo; true; index++)
+     * By creating a FileText having no lines it makes sure that
+     * fileText.size() returning zero size.
+     * This will keep the for loop intact.
+     */
+    @Test
+    public void testKeepForLoopIntact() throws Exception {
+
+        final DefaultConfiguration checkConfig = createModuleConfig(OrderedPropertiesCheck.class);
+        final OrderedPropertiesCheck check = new OrderedPropertiesCheck();
+        check.configure(checkConfig);
+        final String fileName =
+                getPath("InputOrderedProperties2EmptyValue.properties");
+        final File file = new File(fileName);
+        final FileText fileText = new FileText(file, Collections.emptyList());
+        final SortedSet<LocalizedMessage> messages = check.process(file, fileText);
+
+        assertEquals("Wrong messages count: " + messages.size(),
+                1, messages.size());
+    }
+
+    @Test
+    public void testFileExtension() {
+
+        final OrderedPropertiesCheck check = new OrderedPropertiesCheck();
+        assertEquals("File extension should be set", ".properties", check.getFileExtensions()[0]);
+    }
+
+    /**
+     * Method generates NoSuchFileException details. It tries to a open file that does not exist.
+     * @param file to be opened
+     * @return localized detail message of {@link NoSuchFileException}
+     */
+    private static String getFileNotFoundDetail(File file) {
+        // Create exception to know detail message we should wait in
+        // LocalisedMessage
```

Comment:
please make 2lines comment as single line, you have enough space

Reply:
done